build/common.sh

```diff
@@ -107,28 +107,35 @@ readonly KUBE_APISERVER_BASE_IMAGE="${KUBE_APISERVER_BASE_IMAGE:-$KUBE_GORUNNER_
 readonly KUBE_CONTROLLER_MANAGER_BASE_IMAGE="${KUBE_CONTROLLER_MANAGER_BASE_IMAGE:-$KUBE_GORUNNER_IMAGE}"
 readonly KUBE_SCHEDULER_BASE_IMAGE="${KUBE_SCHEDULER_BASE_IMAGE:-$KUBE_GORUNNER_IMAGE}"
 readonly KUBE_PROXY_BASE_IMAGE="${KUBE_PROXY_BASE_IMAGE:-$KUBE_BASE_IMAGE_REGISTRY/distroless-iptables:$__default_distroless_iptables_version}"
+readonly KUBE_PROXY_WINDOWS_BASE_IMAGE="${KUBE_PROXY_WINDOWS_BASE_IMAGE:-mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v0.1.0}"
 readonly KUBECTL_BASE_IMAGE="${KUBECTL_BASE_IMAGE:-$KUBE_GORUNNER_IMAGE}"
 
 # This is the image used in a multi-stage build to apply capabilities to Docker-wrapped binaries.
 readonly KUBE_BUILD_SETCAP_IMAGE="${KUBE_BUILD_SETCAP_IMAGE:-$KUBE_BASE_IMAGE_REGISTRY/setcap:$__default_setcap_version}"
 
-# Get the set of master binaries that run in Docker (on Linux)
+# Get the set of master binaries that run in Docker (on Linux), or as Host Processes on Windows.
 # Entry format is "<binary-name>,<base-image>".
-# Binaries are placed in /usr/local/bin inside the image.
+# Binaries are placed in /usr/local/bin inside the image (Linux), or C:\bin (Windows).
```

Comment:
```suggestion
# Binaries are placed in /usr/local/bin inside the image (Linux), or c:\hpc (Windows).
```

Reply:
> Do you suggest we just copy the kube-proxy binary and other scripts to `C:\` in the Dockerfile?

No I don't think so.  I think its just a bit confusing... If you start a HPC pod it won't be at `c:\bin` but at `c:\hpc\bin`, maybe there is a way to word it so its clearer but I am not sure.   If I read the dockerfile correctly its actually at `c:/hpc/kube-proxy/kube-proxy.exe`?